src/models/methods/index.ts

```diff
@@ -26,6 +28,11 @@ type Request = // account methods
              | AccountTxRequest
              | GatewayBalancesRequest
              | NoRippleCheckRequest
+               // payment channel methods
+             | ChannelAuthorizeRequest
```

Comment:
`ChannelAuthorizeRequest` and `ChannelVerifyRequest`  go here, but the responses should go under `type Response`

Reply:
Oops, went over my head haha 😅